howtos/create_installer_packages.rst

```diff
@@ -0,0 +1,115 @@
+- The package_info method use the new “self.env_info” object. With “self.env_info” the package can declare environment variables that will be setted with the “virutalenv” generator.
+
+
+The “self.env_info” variable can also be useful if a package tool depends on another tool.
+Take a look to the MinGW conanfile.py recipe (https://www.conan.io/source/mingw_installer/0.1/lasote/testing):
+
+
+.. code-block:: python
+
+   class MingwinstallerConan(ConanFile):
+       name = "mingw_installer"
+       version = "0.1"
+       license = "MIT"
+       url = "http://github.com/lasote/conan-mingw-installer"
+       settings = {"os": ["Windows"]}
+       options = {"threads": ["posix", "win32"],
+                  "exception": ["dwarf2", "sjlj", "seh"], 
+                  "arch": ["x86", "x86_64"],
+                  "version": ["4.8", "4.9"]}
+       default_options = "exception=sjlj", "threads=posix", "arch=x86_64", "version=4.9"
+   
+       def config(self):
+           self.requires.add("7z_installer/0.1@lasote/testing", private=True)
+           …
+      
+       def build(self):
+           ...
+           
+           tools.download(files[keychain], "file.7z")
+           env = ConfigureEnvironment(self)
+           self.run("%s && 7z x file.7z" % env.command_line)
+       
+       def package(self):
+           self.copy("*", dst="", src="mingw32")
+           self.copy("*", dst="", src="mingw64")
+   
+       def package_info(self):
+           self.env_info.path.append(os.path.join(self.package_folder, "bin"))
+           self.env_info.CXX = os.path.join(self.package_folder, "bin", "g++.exe")
+           self.env_info.CC = os.path.join(self.package_folder, "bin", "gcc.exe")
+
+
+In the config method we are adding a require to another package, the 7z_installer that will use to unzip the mingw installers (with 7z compression).
+
+In the build method we are downloading the right MinGW installer and using the helper 
+``ConfigureEnvironment``. This helper will provide us a string with a command to set the environment variables. That means that the 7z executable will be in the path, because the 7z_installer dependency declares the “bin” folder in it’s “package_info” method.
+
+In the package_info method we are declaring CC and CXX variables, used by CMake, autotools etc, to locate the compiler for C/C++ respectively. 
+Also we are appending to “path” variable the bin folder, so we can invoke gcc, g++, make and other tools in the command line using the virtualenv generator when we execute the “activate” script.
```

Comment:
I miss the consumer use case. I would add examples of consuming and changing MinGW and CMake versions with this approach.